src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -429,6 +434,54 @@ def generate_request_payload(configuration, location, public_key, tags):
     return cc
 
 
+def get_kubeconfig_node_dict(kube_config=None):
+    if kube_config is None:
+        kube_config = os.getenv('KUBECONFIG') if os.getenv('KUBECONFIG') else os.path.join(os.path.expanduser('~'), '.kube', 'config')
+    try:
+        kubeconfig_data = config.kube_config._get_kube_config_loader_for_yaml_file(kube_config)._config
+    except Exception as ex:
+        telemetry.set_user_fault()
+        telemetry.set_exception(exception=ex, fault_type=consts.Load_Kubeconfig_Fault_Type,
+                                summary='Error while fetching details from kubeconfig')
+        raise CLIError("Error while fetching details from kubeconfig." + str(ex))
+    return kubeconfig_data
+
+
+def check_aks_cluster(kube_config, kube_context):
+    config_data = get_kubeconfig_node_dict(kube_config=kube_config)
```

Comment:
Can we now use this method in get_user_aad_details method instead of calling kubectl config view?.

Reply:
Yes, have already done that. Added that in your private repo pr.

Reply:
okay cool thanks did not realize it's the public repo PR.